server/database/firestore.js

```diff
@@ -1,8 +1,21 @@
 import admin from 'firebase-admin';
-import { serviceAccount } from './keys';
+import fs from 'fs';
+import path from 'path';
+import url from 'url';
+import { isDevelopment } from '../utils/util';
 
-export const firebaseAdmin = admin.initializeApp({
-  credential: admin.credential.cert(serviceAccount),
-});
+let firebaseAuth;
+if (isDevelopment()) {
```

Comment:
Ok so looks like this change requires that we have the key loaded only if we're in development. If in production, the app will get the key that's stored in gcp?